inc/migrate-post.php

```diff
@@ -130,15 +130,17 @@ public function count_tags( $input ) {
 	}
 
 	/**
-	 * Extract data that will be converted to YAML front matter
+	 * Extract data that will be converted to YAML front matter.
+	 * Use html_entity_decode() because Hugo templates automatically encode.
 	 *
 	 * @param WP_Post $post
 	 * @return array
 	 */
 	public function extract_front_matter( $post ) {
+		$author_name = get_the_author_meta( 'display_name', intval( $post->post_author ) );
```

Comment:
since display_name isn't required to create a user, should prbly check this field exists before inserting in $data array, and have a fallback (maybe empty string?)